ide/gototest/src/org/netbeans/modules/gototest/GotoOppositeAction.java

```diff
@@ -126,26 +124,38 @@ public void performAction() {
 
                 @Override
                 public void run() {
-                    FileType currentFileType = getCurrentFileType();
-                    if(currentFileType == FileType.NEITHER) {
-                        StatusDisplayer.getDefault().setStatusText(Bundle.No_Test_Or_Tested_Class_Found());
+                    LocatorResult opposites;
+
+                    try {
+                        opposites = TestOppositesLocator.getDefault().findOpposites(fo, caretOffset).get();
+                    } catch (InterruptedException | ExecutionException ex) {
+                        Exceptions.printStackTrace(ex);
+                        return ;
+                    }
+
+                    if (opposites.getErrorMessage() != null) {
+                        StatusDisplayer.getDefault().setStatusText(opposites.getErrorMessage());
```

Comment:
Can the execution reach here in headless mode ? If so, I would skip the dialog.

Reply:
I am afraid it won't be possible most probably.